src/index.jsx

```diff
@@ -25,6 +29,26 @@ const App = () => (
   </Provider>
 );
 
+const Unauthorized = () => (
+  <div>
+    You are not authorized to access the gradebook for this course.
```

Comment:
Maybe the message can be improved to say `You are not enrolled or authorized to access the gradebook for this course`?